scripts/india_nhm/base/data_cleaner.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pandas as pd
+
+INDIA_ISO_CODES = {
```

Comment:
Can these codes be merged into 
scripts/india/geo/states.py and then use get_state_name_to_iso_code_mapping(state_name)?

Reply:
I think Thej is working on creating a geos folder. Not sure about that. If he is not, I can create it and implement this.